controllers/provisioners/eks/update_test.go

```diff
@@ -109,7 +109,12 @@ func TestUpdateWithDriftRotationPositive(t *testing.T) {
 		},
 		ClusterNodes: nodes,
 		Cluster: &eks.Cluster{
-			Version: aws.String("1.15"),
+			CertificateAuthority: &eks.Certificate{
```

Comment:
Can we create a MockEksCluster function with default values and then have a more tidy declaration? e.g.
```golang
Cluster: MockEksCluster()
```
you can add any element we want to control as arguments to this function.
This will remove all the dupe code